site/_posts/2022-12-22-manageiq-oparin-ga-announcement.md

```diff
@@ -0,0 +1,98 @@
+    * `/custom_button_sets` reorder [[#1146]](https://github.com/ManageIQ/manageiq-api/issues/1146)
+    * `/host_initiators` delete [[#1157]](https://github.com/ManageIQ/manageiq-api/issues/1157)
+    * `/physical_servers` decommission_server, recommission_server [[#1144]](https://github.com/ManageIQ/manageiq-api/issues/1144)
+    * `/physical_storages` validate [[#1163]](https://github.com/ManageIQ/manageiq-api/issues/1163)
+    * `/vms` set_description [[#1120]](https://github.com/ManageIQ/manageiq-api/issues/1120)
+  * Fixes
+    * `/widgets` Fixed generate_content to now return a task_id [[#1165]](https://github.com/ManageIQ/manageiq-api/issues/1165)
+    * `/volume_mappings` Fixed issue where configuration button for volume mappings was missing [[#1169]](https://github.com/ManageIQ/manageiq-api/issues/1169)
+* Build
+  * Default to PostgreSQL 13 [[#20662]](https://github.com/ManageIQ/manageiq/issues/20662)
+  * [Podified] operator-sdk 1.x support [[#624]](https://github.com/ManageIQ/manageiq-pods/issues/624)
+  * Remove the need to deliver NodeJS in production [[#8300]](https://github.com/ManageIQ/manageiq-ui-classic/issues/8300)
+  * Move ansible python module installation from post script to a standalone rpm [[#423]](https://github.com/ManageIQ/manageiq-appliance-build/issues/423)
+* Developer
+  * Complete Transition from AvailabilityMixin to SupportsFeatureMixin [[#21179]](https://github.com/ManageIQ/manageiq/issues/21179)
+  * [UI] Continue conversion of HAML tables to react Carbon tables [[#8124]](https://github.com/ManageIQ/manageiq-ui-classic/issues/8124)
+  * [UI] Continue to replace angular code [[#8417]](https://github.com/ManageIQ/manageiq-ui-classic/issues/8417)
+  * [UI] Continue conversion of forms to React [[#8441]](https://github.com/ManageIQ/manageiq-ui-classic/issues/8441)
+  * Support Ruby 3.0 [[#21536]](https://github.com/ManageIQ/manageiq/issues/21536)
+  * Upgrade to Rails 6.1 [[#21516]](https://github.com/ManageIQ/manageiq/issues/21516)
+  * Upgrade ansible-runner to 2.3.x [[#21734]](https://github.com/ManageIQ/manageiq/issues/21734)
+
+**IMPORTANT**: The following are potentially breaking changes and deprecations.
+
+* \*\***BREAKING**\*\*
+  * Stop logging to files in production deployments [[#21177]](https://github.com/ManageIQ/manageiq/pull/21177)
+    * Logs will go to `journald` on appliances and to STDOUT on podified deployments
+  * Don't automatically initialize appliances on first-boot [[#21878]](https://github.com/ManageIQ/manageiq/issues/21878)
+    * On first-boot of the first appliance, if the appliance is meant to be a database appliance, then a database volume will need to be added before initializing the database.
+    * On first-boot of every other appliance, the appliance can be directly connected to the first appliance without need for "un-initialization".
```

Comment:
@agrare I'm not sure of the wording on these two.

Reply:
How about something like
```
* On first-boot an appliance can be configured as a database appliance or a worker appliance using `appliance_console`
* Worker appliances do no need a second disk and can be connected directly to the database appliance
```

Reply:
Updated.